vassal-app/src/main/java/VASSAL/build/module/map/BoardPicker.java

```diff
@@ -175,6 +175,7 @@ public void addTo(Buildable b) {
     for (Board board : possibleBoards) {
       board.setMap(map);
     }
+    // TODO b instanceof Translatable is always true
```

Comment:
Intellij believes that b instanceof Translatable is always true, but I don't see it. Translatable extends Buildable and b is Buildable. All Buildables in Vassal may currently be Translatable, but that isn't guaranteed to be true in the future.